src/com/massivecraft/massivegates/entity/MConf.java

```diff
@@ -72,5 +73,9 @@
 	public int floodFillLimit = 200;
 	
 	public int lineOfSightLimit = 100;
+	
+	public Material visFrame = Material.JACK_O_LANTERN;
+	public Material visContent = Material.GLOWSTONE;
+	public Material visPower = Material.STATIONARY_LAVA;
 
```

Comment:
May I suggest more expressive names such as "visualizationBlockFrame", "visualizationBlockContent", "visualizationBlockPower"?

Reply:
Sounds good to me